# This file is distributed under the same license as the Django package.
#
# Translators:
# Ruben Harutyunov <rharutyunov@mail.ru>, 2018
msgid ""
msgstr ""
"Project-Id-Version: django\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-05-01 23:33+0400\n"
"PO-Revision-Date: 2018-11-11 20:11+0000\n"
"Last-Translator: Ruben Harutyunov <rharutyunov@mail.ru>\n"
"Language-Team: Armenian (http://www.transifex.com/django/django/language/"
"hy/)\n"
"Language: hy\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\admin.py:49
msgid "Personal info"
msgstr "Անձնական տվյալներ"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\admin.py:51
msgid "Permissions"
msgstr "Իրավունքներ"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\admin.py:62
msgid "Important dates"
msgstr "Կարևոր ամսաթվեր"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\admin.py:158
#, python-format
msgid "%(name)s object with primary key %(key)r does not exist."
msgstr "%(key)r հիմնական բանալով %(name)s օբյեկտ գոյություն չունի։"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\admin.py:170
msgid "Password changed successfully."
msgstr "Գաղտնաբառը հաջողությամբ փոխվեց։"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\admin.py:191
#, python-format
msgid "Change password: %s"
msgstr "Փոխել գաղտնաբառը․ %s"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\apps.py:16
msgid "Authentication and Authorization"
msgstr "Նույնականացում և Լիազորում"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\base_user.py:60
msgid "password"
msgstr "գաղտնաբառ"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\base_user.py:61
msgid "last login"
msgstr "վերջին մուտք"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\forms.py:41
msgid "No password set."
msgstr "Գաղտնաբառը նշված չէ։"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\forms.py:49
msgid "Invalid password format or unknown hashing algorithm."
msgstr "Գաղտնաբառի սխալ ֆորմատ, կամ անհայտ հեշավորման ալգորիթմ։"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\forms.py:99
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\forms.py:393
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\forms.py:471
#, fuzzy
#| msgid "The two password fields didn't match."
msgid "The two password fields didn’t match."
msgstr "Երկու գաղտնաբառերը չեն համապատասխանում իրար։"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\forms.py:102
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\forms.py:180
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\forms.py:215
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\forms.py:475
msgid "Password"
msgstr "Գաղտնաբառ"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\forms.py:108
msgid "Password confirmation"
msgstr "Գաղտնաբառը նորից"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\forms.py:111
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\forms.py:486
msgid "Enter the same password as before, for verification."
msgstr "Մուտքագրեք հին գաղտնաբառը, ստուգման համար։"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\forms.py:182
msgid ""
"Raw passwords are not stored, so there is no way to see this user’s "
"password, but you can change the password using <a href=\"{}\">this form</a>."
msgstr ""

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\forms.py:222
#, python-format
msgid ""
"Please enter a correct %(username)s and password. Note that both fields may "
"be case-sensitive."
msgstr ""
"Մուտքագրեք ճիշտ %(username)s և գաղտնաբառ։ Երկու դաշտերն էլ տառաշարազգայուն "
"են։"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\forms.py:225
msgid "This account is inactive."
msgstr "Այս օգտագործողը ակտիվ չէ։"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\forms.py:290
msgid "Email"
msgstr "Email"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\forms.py:396
msgid "New password"
msgstr "Նոր գաղտնաբառ"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\forms.py:402
msgid "New password confirmation"
msgstr "Նոր գաղտնաբառը նորից"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\forms.py:439
msgid "Your old password was entered incorrectly. Please enter it again."
msgstr "Հին գաղտնաբառը սխալ է։ Մուտքագրեք նորից։"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\forms.py:443
msgid "Old password"
msgstr "Հին գաղտնաբառ"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\forms.py:483
msgid "Password (again)"
msgstr "Գաղտնաբառ (նորից)"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:344
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:437
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:527
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:622
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:682
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:724
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:782
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:837
msgid "algorithm"
msgstr "ալգորիթմ"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:345
msgid "iterations"
msgstr "իտերացիաներ"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:346
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:443
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:529
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:626
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:683
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:725
msgid "salt"
msgstr "աղ"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:347
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:444
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:627
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:684
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:726
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:783
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:838
msgid "hash"
msgstr "հեշ"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:438
msgid "variety"
msgstr ""

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:439
msgid "version"
msgstr ""

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:440
msgid "memory cost"
msgstr ""

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:441
msgid "time cost"
msgstr "տևողություն"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:442
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:625
msgid "parallelism"
msgstr ""

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:528
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:623
msgid "work factor"
msgstr "աշխատանքային ֆակտոր"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:530
msgid "checksum"
msgstr "checksum"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\hashers.py:624
msgid "block size"
msgstr ""

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:62
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:116
msgid "name"
msgstr "անուն"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:66
msgid "content type"
msgstr "պարունակության տիպ"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:68
msgid "codename"
msgstr "կոդային անուն"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:73
msgid "permission"
msgstr "իրավունքնե"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:74
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:119
msgid "permissions"
msgstr "իրավունքներ"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:126
msgid "group"
msgstr "խումբ"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:127
#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:258
msgid "groups"
msgstr "խմբեր"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:249
msgid "superuser status"
msgstr "սուպերօգտագործողի կարգավիճակ"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:252
msgid ""
"Designates that this user has all permissions without explicitly assigning "
"them."
msgstr ""
"Ցույց է տալիս, որ օգտագործողը ունի բոլոր իրավունքները, առանց նրանց նշման։"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:261
msgid ""
"The groups this user belongs to. A user will get all permissions granted to "
"each of their groups."
msgstr ""
"Խումբը, որին պատկանում է օգտագործողը։ Օգտագործողը ստանում է իր խմբին տրված "
"բոլոր իրավունքները։"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:269
msgid "user permissions"
msgstr "օգտագործողի իրավունքները"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:271
msgid "Specific permissions for this user."
msgstr "Օգտագործողի հատուկ իրավունքները։"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:345
msgid "username"
msgstr "օգտագործողի անուն"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:349
msgid "Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only."
msgstr ""

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:353
msgid "A user with that username already exists."
msgstr "Այդ անունով օգտագործող արդեն գոյություն ունի։"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:356
msgid "first name"
msgstr "անուն"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:357
msgid "last name"
msgstr "ազգանուն"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:358
msgid "email address"
msgstr "email հասցե"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:360
msgid "staff status"
msgstr "անձնակազմի կարգավիճակ"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:362
msgid "Designates whether the user can log into this admin site."
msgstr ""
"Ցույց է տալիս, թե արդյոք օգտագործողը կարոզ է մուտք գործել ադմինիստրավորման "
"բաժին։"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:365
msgid "active"
msgstr "ակտիվ"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:368
msgid ""
"Designates whether this user should be treated as active. Unselect this "
"instead of deleting accounts."
msgstr ""
"Ցույց է տալիս, թե արդյոք օգտագործողին կարելի է համարել ակտիվ։ Ապընտրեք, "
"օգտագործողին հեռացնելու փոխարեն։"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:372
msgid "date joined"
msgstr "միացել է"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:381
msgid "user"
msgstr "օգտագործող"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\models.py:382
msgid "users"
msgstr "օգտագործողներ"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\password_validation.py:111
#, python-format
msgid ""
"This password is too short. It must contain at least %(min_length)d "
"character."
msgid_plural ""
"This password is too short. It must contain at least %(min_length)d "
"characters."
msgstr[0] ""
"Գաղտնաբառը շատ կարճ է։ Այն պետք է պարունակի ամենաքիչը %(min_length)d նիշ։"
msgstr[1] ""
"Գաղտնաբառը շատ կարճ է։ Այն պետք է պարունակի ամենաքիչը %(min_length)d նիշ։"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\password_validation.py:123
#, python-format
msgid "Your password must contain at least %(min_length)d character."
msgid_plural "Your password must contain at least %(min_length)d characters."
msgstr[0] "Գաղտնաբառը պետք է պարունակի ամենաքիչը %(min_length)d նիշ։"
msgstr[1] "Գաղտնաբառը  պետք է պարունակի ամենաքիչը %(min_length)d նիշ։"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\password_validation.py:206
#, python-format
msgid "The password is too similar to the %(verbose_name)s."
msgstr "Գաղտնաբառը շատ նման է %(verbose_name)s֊ին։"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\password_validation.py:213
#, fuzzy
#| msgid ""
#| "Your password can't be too similar to your other personal information."
msgid "Your password can’t be too similar to your other personal information."
msgstr "Գաղտնաբառը չի կարող շատ նման լինել ձեր անձնական ինֆորմացիային։"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\password_validation.py:245
msgid "This password is too common."
msgstr "Գաղտնաբառը շատ տարածված է։"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\password_validation.py:250
#, fuzzy
#| msgid "Your password can't be a commonly used password."
msgid "Your password can’t be a commonly used password."
msgstr "Գաղտնաբառը չպետք է լինի տարածված գաղտնաբառերից մեկը։"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\password_validation.py:261
msgid "This password is entirely numeric."
msgstr "Գաղտնաբառը բաղկացած է միայն թվերից։"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\password_validation.py:266
#, fuzzy
#| msgid "Your password can't be entirely numeric."
msgid "Your password can’t be entirely numeric."
msgstr "Գաղտնաբառը չպետք է բաղկացած լինի միայն թվերից։"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\templates\registration\password_reset_subject.txt:2
#, python-format
msgid "Password reset on %(site_name)s"
msgstr "Գաղտնաբառի փոփոխում %(site_name)s կայքում։"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\validators.py:12
msgid ""
"Enter a valid username. This value may contain only unaccented lowercase a-z "
"and uppercase A-Z letters, numbers, and @/./+/-/_ characters."
msgstr ""

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\validators.py:22
msgid ""
"Enter a valid username. This value may contain only letters, numbers, and "
"@/./+/-/_ characters."
msgstr ""

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\views.py:163
msgid "Logged out"
msgstr "Դուք դուրս եք եկել"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\views.py:222
msgid "Password reset"
msgstr "Գաղտնաբառի փոփոխում"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\views.py:249
msgid "Password reset sent"
msgstr "Գաղտնաբառի փոփոխման հղումն ուղարկված է"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\views.py:259
msgid "Enter new password"
msgstr "Մուտքագրեք նոր գաղտնաբառը"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\views.py:331
msgid "Password reset unsuccessful"
msgstr "Գաղտնաբառի փոփոխումը չի ավարտվել հաջողությամբ"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\views.py:340
msgid "Password reset complete"
msgstr "Գաղտնաբառի փոփոխումը ավարտված է"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\views.py:352
msgid "Password change"
msgstr "Գաղտնաբառի փոփոխում"

#: .\venv\lib\python3.11\site-packages\django\contrib\auth\views.py:375
msgid "Password change successful"
msgstr "Գաղտնաբառի փոփոխումը ավարտվել է հաջողությամբ"
